openedx/features/qverse_features/registration/tasks.py

```diff
@@ -0,0 +1,55 @@
+"""
+Qverse registration application celery tasks.
+"""
+from celery.task import task
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from edx_ace import ace
+from edx_ace.recipient import Recipient
+
+from openedx.core.djangoapps.ace_common.template_context import get_base_template_context
+from openedx.core.djangoapps.theming.helpers import get_current_site
+from openedx.core.lib.celery.task_utils import emulate_http_request
+from openedx.features.qverse_features.registration.message_types import RegistrationNotification
+from openedx.features.qverse_features.registration.utils import get_user_by_regno_or_username
+
+
+LOGGER = get_task_logger(__name__)
+ACE_ROUTING_KEY = getattr(settings, 'ACE_ROUTING_KEY')
```

Comment:
This will raise an exception if there is no value set for `ACE_ROUTING_KEY`.  Is this the required behavior here?

Reply:
Have added default value here.